tests/test_main.py

```diff
@@ -1,6 +1,11 @@
+import mock
+import sys
+
 from validator.main import main
 
-def test_main():
+
+@mock.patch.object(sys, 'stderr')
```

Comment:
Oh, I remember now, yes indeed. Also, it's not a "base test that could be inherited/re-used" elsewhere, so my comment is irrelevant. Thanks.